modules/ROOT/pages/sap/sap-reference-45.adoc

```diff
@@ -0,0 +1,815 @@
+`<sap:execute-queued-remote-function-call>`
+
+
+Executes a BAPI function over a queued Remote Function Call (qRFC). A queued RFC is an extension of a transactional RFC (tRFC) that ensures that individual steps are processed in sequence. To guarantee that multiple Logical Unit of Work (LUWs) transactions are processed in the order specified by the application, tRFC can be serialized using inbound and outbound queues; hence the name queued RFC (qRFC). 
+
+Use qRFC to extend tRFC and define the processing sequence.  qRFC is recommended if you want to guarantee that several transactions are processed in a predefined order.
+
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use | | x
+| Key a| String |  The name of the BAPI function to execute |  | x
+| Content a| Binary |  The BAPI function to execute. |  `#[payload]` | 
+| Transaction Id a| String |  The ID that identifies an RFC so it runs only once |  | 
+| Queue Name a| String |  The name of the queue on which the RFC executes |  | x
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  The type of joining action that operations can take regarding transactions |  `JOIN_IF_POSSIBLE` | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  | 
+|===
+
+
+=== For Configurations
+* <<outbound-config>> 
+
+==== Throws
+
+* SAP:BEGIN_TRANSACTION_ERROR 
+* SAP:COMMIT_TRANSACTION_ERROR 
+* SAP:CONFIG_ERROR 
+* SAP:CONFIRM_TRANSACTION_ERROR 
+* SAP:CONNECTION_ERROR 
+* SAP:CONNECTIVITY 
+* SAP:EXECUTION_ERROR 
+* SAP:FIELD_NOT_FOUND_ERROR 
+* SAP:INVALID_CREDENTIALS_ERROR 
+* SAP:INVALID_HOST_ERROR
+* SAP:MISSING_BUSINESS_OBJECT_ERROR 
+* SAP:RETRY_EXHAUSTED 
+* SAP:ROLLBACK_TRANSACTION_ERROR 
+* SAP:TID_CREATION_ERROR 
+* SAP:UNKNOWN 
+* SAP:XML_PARSING_ERROR  
+
+
+[[executeSynchronousRemoteFunctionCall]]
+=== Execute BAPI function over sRFC
+`<sap:execute-synchronous-remote-function-call>`
+
+
+Executes a BAPI function over a synchronous Remote Function Call (sRFC). A synchronous RFC requires that both the client and server systems are available at the time of communication or data transfer. A synchronous RFC is the most common type and is used when a result is required immediately after the sRFC is executed.  
+
+Use sRFC when communication between source and target systems requires acknowledgments. The resources of the source system wait at the target system and ensure that the message or data is delivered with an acknowledgment (ACKD) and that the data is consistent and reliable for communication.  
+
+If the target system is not available, the source system resources wait until the target system is available. This may lead to the processes of the source system to go into Sleep/RFC/CPIC Mode at the target systems and block the resources.  
```

Comment:
```suggestion
If the target system is not available, the source system resources wait until the target system is available. This may cause the processes of the source system to go into Sleep/RFC/CPIC Mode at the target systems and block the resources.  
```